modules/axtask/src/task.rs

```diff
@@ -39,9 +50,26 @@ pub struct TaskInner {
     entry: Option<*mut dyn FnOnce()>,
     state: AtomicU8,
 
+    /// CPU affinity mask.
+    cpumask: SpinNoIrq<CpuMask>,
+
+    /// Mark whether the task is in the wait queue.
     in_wait_queue: AtomicBool,
+
+    /// Used to indicate whether the task is running on a CPU.
+    on_cpu: AtomicBool,
```

Comment:
Maybe it is unnecessary when `SMP` feature isn't enabled?

Reply:
I think it's better to add `#[cfg(feature = "smp")]` here, 
althrough polling on `on_cpu` will never happen in a singal core system

Reply:
@equation314

Reply:
> I think it's better to add `#[cfg(feature = "smp")]` here, althrough polling on `on_cpu` will never happen in a singal core system

OK